src/OrderFormAcceptanceTests.TestData/Address.cs

```diff
@@ -27,7 +27,6 @@ public Address Generate()
             var randomAddress = faker.Address;
             return new Address()
             {
-                AddressId = faker.Random.Number(),
                 Line1 = randomAddress.StreetAddress(),
```

Comment:
Do we use Address.Line2 or Address.Line3 for anything in the tests?

Reply:
not yet, but the application isn't pulling the correct details from OAPI yet, and at that point we might want to store all the lines from the DB and assert they are correct on the UI

Reply:
seems reasonable